kiwi/tasks/system_build.py

```diff
@@ -49,6 +51,15 @@
         when using obs:// repos resolve them using the SUSE internal
         buildservice. This only works if access to SUSE's internal
         buildservice is granted
+    --set-container-derived-from=<uri>
+        overwrite the source location of the base container
+        for the selected image type. The setting is only effective
+        if the configured image type is setup with an initial
+        derived_from reference
+    --set-container-tag=<name>
+        overwrite the container tag in the container configuration.
```

Comment:
This is because it is expected to be a configurable value form the OBS project configuration? I have difficulties to see the use case.